en/contribute/unit_tests.md

```diff
@@ -7,11 +7,13 @@ The full list of unit tests can be found in [UnitTestList.cc](https://github.com
 To run unit tests:
 1. Build in `debug` mode with `UNITTEST_BUILD` definition. 
 1. Copy the **deploy/qgroundcontrol-start.sh** script in the **debug** directory
-1. Run *all* unit tests from the command line using the `--unittest` command line option:
+1. Run *all* unit tests from the command line using the `--unittest` command line option. 
```

Comment:
@DonLakeFlyer This is where you were talking about platform specific instructions. I have now updated to make it clear that the `--unittest` option is generic but the example is Linux only.

I think it is useful to include the Linux option because I would not have know how to run this. I also tried to run this on windows on Daily build using `D:\Program Files (x86)\QGroundControl>QGroundControl.exe --unittest` with no luck. Is this the wrong approach, or just that I am using the wrong build? How about for Mac - is that started in the same way as Linux.

I'm going to merge this now and update the readme. However will update this part further if needed - I think it would be good to clarify:
1. Unit tests built into which versions by default? Just your own local builds or also developer build
2. How to run tests on all platforms.

Cheers
H